.drone.yml

```diff
@@ -28,6 +28,14 @@ pipeline:
     db_host: ${DB_HOST}
     git_reference: ${CORE_BRANCH}
 
+  php-cs-fixer:
```

Comment:
This runs in every drone job.
Do we want to add:
```
    when:
      matrix:
        TEST_SUITE: phpunit
```
so that at least it only runs in each of the ``phpunit`` jobs?